servers/src/main/java/tachyon/master/next/filesystem/FileSystemMaster.java

```diff
@@ -272,6 +275,28 @@ public boolean isDirectory(long id) {
     }
   }
 
+  /**
+   * Gets the list of block info of an InodeFile determined by path.
+   *
+   * @param path path to the file
+   * @return The list of the block info of the file
+   * @throws InvalidPathException when the path is invalid
+   * @throws FileDoesNotExistException when the file does not exist
+   */
+  public List<BlockInfo> getBlockInfoList(TachyonURI path)
```

Comment:
Shouldn't the fs matser return FileBlockInfo? And, why can't this re-use getFileBlockInfoList()?